### PR TITLE
Use OSX-compatible sed command.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: packages/package_sicer_1_1/tool_dependencies.xml [hunk 1]
```diff
@@ -8,8 +8,8 @@
                     <source_directory>.</source_directory>
                     <destination_directory>$INSTALL_DIR</destination_directory>
                 </action>
-                <action type="shell_command">sed -i 's#PATHTO=.\+#PATHTO=$INSTALL_DIR#' $INSTALL_DIR/SICER/*.sh</action>
-                <action type="shell_command">sed -i 's#SICER=.\+#SICER=$INSTALL_DIR/SICER#' $INSTALL_DIR/SICER/*.sh</action>
+                <action type="shell_command">sed -i.bak 's#PATHTO=.\+#PATHTO=$INSTALL_DIR#' $INSTALL_DIR/SICER/*.sh</action>
+                <action type="shell_command">sed -i.bak 's#SICER=.\+#SICER=$INSTALL_DIR/SICER#' $INSTALL_DIR/SICER/*.sh</action>
```
Comment (nsoranzo) [inline packages/package_sicer_1_1/tool_dependencies.xml]: I don't have a Mac to test, but according to https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/sed.1.html a better fix may be

```
sed -i -e 's#PATHTO=.\+#PATHTO=$INSTALL_DIR#' $INSTALL_DIR/SICER/*.sh
```

and similarly for the second line.

Comment (bgruening) [inline packages/package_sicer_1_1/tool_dependencies.xml]: @davebx can you confirm this one?

Comment (bgruening) [inline packages/package_sicer_1_1/tool_dependencies.xml]: @natefoo @davebx anyone with a Mac ... what is the best way to do this.

**Comment by martenson** [comment]
ping @davebx - can you please follow up

**Comment by bgruening** [comment]
ping @davebx :)

**Comment by davebx** [comment]
SYN/ACK @bgruening

**Comment by bgruening** [comment]
Thanks @davebx!
:+1:

**Pull request merged**